tests/vc_navier_stokes/SetFluidProperties.cpp

```diff
@@ -0,0 +1,148 @@
+// ---------------------------------------------------------------------
+//
+// Copyright (c) 2018 - 2019 by the IBAMR developers
+// All rights reserved.
+//
+// This file is part of IBAMR.
+//
+// IBAMR is free software and is distributed under the 3-clause BSD
+// license. The full text of the license can be found in the file
+// COPYRIGHT at the top level directory of IBAMR.
+//
+// ---------------------------------------------------------------------
+
+// Filename: SetFluidProperties.cpp
```

Comment:
@drwells, I think we are omitting this material from source code files now --- is that right?

Reply:
Yes.